Avoid an issue on microblaze where the symver attribute is not available.

diff --git a/lib/crypt-port.h b/lib/crypt-port.h
index 3fa5222..80c3d91 100644
--- a/lib/crypt-port.h
+++ b/lib/crypt-port.h
@@ -208,7 +208,7 @@ extern size_t strcpy_or_abort (void *dst, size_t d_size, const void *src);
 
 /* Starting with GCC 10, we can use the symver attribute, which is also
    needed at the point we decide to enable link-time optimization.  */
-# if __GNUC__ >= 10
+# if __GNUC__ >= 10 && ! defined (__MICROBLAZE__)
 
 /* Set the symbol version for EXTNAME, which uses INTNAME as its
    implementation.  */
